README.rst

```diff
@@ -74,6 +74,29 @@ US/Canada, a short code for the UK and an alphanumertic originator for everywher
     originators.44 = short:12345
     originators.default = alpha:Matrix
 
+Docker
+======
+
+A Dockerfile is provided for sydent. To use it, run ``docker build -t sydent .`` in a sydent checkout.
+To run it, use ``docker run --env=SYDENT_SERVER_NAME=my-sydent-server --mount type=volume,source=sydent-data,destination=/data sydent``.
+
+Environment variables
+---------------------
+
+.. warning:: These variables are only taken into account at the first start and written to the configuration file.
+
++--------------------+-----------------+-----------------------+
+| Variable Name      | Sydent default  | Docker default        |
```

Comment:
What do you mean by "Docker default" here? I presume that it's the default location of this file within the docker container. If so, perhaps `Container filepath` may be clearer?

Also, will these paths change if `destination=` in the above command is changed?